src/collections/posts/installing-the-mod-guide-necessary-for-any-custom-songs.md

```diff
@@ -0,0 +1,72 @@
+section: speciality
+category: guide
+publish: 2024-03-22T00:00:00.000Z
+homepageText: null
+image: /static/uploads/posts/beat-saber-square.jpg
+showInPostListing: false
+---
+# If you haven’t played custom songs yet and are just getting started with Beat Saber, we’re excited for you!
+
+## You’re about to experience a whole WORLD of new content and the ability to experience “inside” your favorite music like only VR allows! It’s really a magical experience.
+
+The modding experience for computer-based VR (PCVR) and Meta/Oculus Quest standalone VR is a bit different. Be sure to read the right section for your headset!
+
+## PC VR:
+
+While mods are not *required* to play basic custom songs, mods can enhance your gameplay experience and unlock some nifty mapping effects. To get started, you’ll just need to download [**ModAssistant**](https://github.com/bsmg/ModAssistant/releases)
+
+<p align="center">
+    <img src="/uploads/mod-assistant-mods.png">
+</p>
+
+### Installing Mods With Mod Assistant
+
+1. First, [click here](https://github.com/bsmg/ModAssistant/releases) and click on the Latest release of ModAssistant.exe file to download
+
+<p align="center">
+    <img src="/uploads/mod-assistant-github.png">
+</p>
+
+This will download a small `ModAssistant.exe` file that you can place anywhere on your computer for future use and doesn’t require installing.
+
+2. When you first launch Mod Assistant, it will also try to automatically locate where you installed your game. If it can’t find it, you will have to manually locate the folder.
+
+    - Oculus User folder paths tend to be: `C:\Program Files\Oculus\Software\Software\hyperbolic-magnetism-beat-saber\`
+
+    - Steam User folder paths tend to be: `C:\Program Files (x86)\Steam\steamapps\common\Beat Saber\`
+
+3. You’ll be prompted to agree to the terms of use. Please read through these carefully as they’re important, mainly for the health of our wonderful community and relationship with Beat Games.
+
+<p align="center">
+    <img src="/uploads/mod-assistant-agreement.png">
+</p>
+
+4. Once you agree, you’ll be able to click on the “Mods” button. All of the essential or “Core" mods will be pre-selected, but you can add as many other mods as you'd like in addition to the Core mods. When ready you can just click on the “Install or Update” button.
+
+    > ⚠️ IMPORTANT: You will need to have run Beat Saber at least once after installing the game before installing mods.
+
+<p align="center">
+    <img src="/uploads/mod-assistant-mods-install.png">
+</p>
+
+5. That’s it! You can now launch your game and enjoy all the wonderful mods that the community has built! You can always open ModAssistant again later to install more mods. We recommend against installing every mod at first as you’ll likely be overwhelmed by all the new options - and you may run into issues. Instead, take the time to learn about what each mod does. Click on the “Mod Info” button after selecting a mod to read more about it.
```

Comment:
Use of a curly quotation mark. Not sure if that is intended so just pointing that out.